stratum/hal/lib/barefoot/bf_switch.cc

```diff
@@ -26,9 +26,11 @@ namespace barefoot {
 
 BFSwitch::BFSwitch(PhalInterface* phal_interface,
                    BFChassisManager* bf_chassis_manager,
+                   BFPdInterface* bf_pd_interface,
                    const std::map<int, PINode*>& unit_to_pi_node)
     : phal_interface_(CHECK_NOTNULL(phal_interface)),
       bf_chassis_manager_(CHECK_NOTNULL(bf_chassis_manager)),
+      bf_pd_interface_(CHECK_NOTNULL(bf_pd_interface)),
```

Comment:
Even though we alias `ABSL_DIE_IF_NULL` to `CHECK_NOTNULL`, the majority of our code is using `ABSL_DIE_IF_NULL`.

Your call.